template/build/webpack.prod.conf.js

```diff
@@ -43,22 +42,6 @@ var webpackConfig = merge(baseWebpackConfig, {
     new ExtractTextPlugin({
       filename: utils.assetsPath('css/[name].[contenthash].css')
     }),
-  // Analyzes your code and remove unused CSS from resulted file
-	new PurifyCSSPlugin({
```

Comment:
Hi there, Im trying to do the similar case to remove unused CSS from Vue webpack template. I have post css and css @imports in my single component file. When I use what you have done here, i dont see its removing all the unused CSS.